src/client/components/app/ActionButtons/index.tsx

```diff
@@ -1,14 +1,14 @@
 import styled from 'styled-components';
 
 import { Button, Icon, WarningFilledIcon } from '@components/common';
+import { device } from '@themes/default';
 
 const ActionButtonsContainer = styled.div<{ actions: number; direction?: string }>`
   display: flex;
   align-items: center;
-  // grid-template-columns: repeat(${({ actions }) => actions}, 1fr);
   gap: ${({ theme }) => theme.layoutPadding};
 
-  @media (max-width: 1800px) {
+  @media (max-width: 2100px) {
```

Comment:
why not use `device` settings u imported instead of hardcoding pixel width?

Reply:
You are definitely right - replace this. 

Regardless, this isn't doing what I want yet (which is to display action buttons either horizontally or locally based on screen size). Will address in this ticket: [Make Position Table action buttons stack horizontal instead of vertical on smaller screens](https://www.notion.so/debtdao/Make-Position-Table-action-buttons-stack-horizontal-instead-of-vertical-on-smaller-screens-eb98fd48a83647e6940c16fd6c529cb8?pvs=4)